src/System.Drawing.Primitives/tests/PointFTests.cs

```diff
@@ -118,9 +125,29 @@ public void EqualityTest(float x, float y)
         }
 
         [Fact]
-        public void ToStringTest()
+        public static void EqualityTest_NotPointF()
+        {
+            var point = new PointF(0, 0);
+            Assert.False(point.Equals(null));
+            Assert.False(point.Equals(0));
+            Assert.False(point.Equals(new Point(0, 0)));
+        }
+
+        [Fact]
+        public static void GetHashCodeTest()
         {
-            PointF p = new PointF(0, 0);
+            var point = new PointF(10, 10);
+            Assert.Equal(point.GetHashCode(), new PointF(10, 10).GetHashCode());
+            Assert.NotEqual(point.GetHashCode(), new PointF(20, 10).GetHashCode());
+            Assert.NotEqual(point.GetHashCode(), new PointF(10, 20).GetHashCode());
```

Comment:
Technically these NotEqual assertions aren't valid; it'd be perfectly fine for these to end up with the same hashcode.  We can leave these for now, as I agree there's value in, for example, making sure GetHashCode isn't returning a constant, but this is the kind of thing we made need to revisit in the future.